docs/adr/2021-10-23_010-use-direct-chain.md

```diff
@@ -30,6 +30,4 @@ For the time being, until _Plutus Application Framework_ is released, we will im
 
 * Limit Hydra dependencies to [plutus](https://github.com/input-output-hk/plutus) repository
 * Remove all PAB-related code as well as off-chain `Contract`s code and related dependencies
-* An updated architecture diagram:
-
-![](../../hydra-node/images/hydra-architecture-direct.jpg)
+* See [Architecture page](/core-concepts/architecture) for details
```

Comment:
```suggestion
```

Would drop it, as it was pointing to a picture that does not exist and it's not a value-adding consequence. An ADR always may result in a changed architecture :shrug:

Reply:
Fair enough